sdk/iOS/test/MSCoreDataStoreTests.m

```diff
@@ -47,6 +50,54 @@ -(void)testInit
     XCTAssertNotNil(self.store, @"store creation failed");
 }
 
+// Tests MSCoreDataStore Read, Update and Delete operations are case insensitive for the Id column.
+-(void)testReadUpdateDelete_CaseInsensitiveIdOperations
+{
+    NSError *error;
+
+    // Seed data to be inserted in the table
+    NSString * const originalItemId = @"itemid";
```

Comment:
The same holds true for almost all the variables in these tests and this is not the style we've been following to date, so I'd prefer to match current style all else being equal.  If you wanted to define static consts at the top of the file so all these tests could share 2 different cased Ids that would be a cleaner approach.